src/components/books/Book.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ title, author }) => (
+  <>
+    <h2>{title}</h2>
+    <p>{author}</p>
+  </>
```

Comment:
- [ ] kindly consider adding a button for removing a book to meet the below requirement
> Has a button for removing books
> * the button wouldn't remove the books *yet*, this is just a placeholder for now